dockerExtension.ts

```diff
@@ -176,7 +176,7 @@ function registerDockerCommands(azureAccount: AzureAccount): void {
     registerCommand('vscode-docker.container.show-logs', showLogsContainer);
     registerCommand('vscode-docker.container.open-shell', openShellContainer);
     registerCommand('vscode-docker.container.remove', removeContainer);
-    registerCommand('vscode-docker.image.build', async function (this: IActionContext): Promise<void> { await buildImage(this); });
+    registerCommand('vscode-docker.image.build', async function (this: IActionContext, item: vscode.Uri | undefined): Promise<void> { await buildImage(this, item); });

```

Comment:
Just asking for reference are the following correct? 
Call from explorer context menu: uri not passed
Editor context menu: Uri passed 
Command palette: Uri not passed

Reply:
> call from explorer context menu: uri not passed

URI passed